bigchaindb/tendermint/commands.py

```diff
@@ -44,6 +50,18 @@ def start():
             '--consensus.create_empty_blocks=false',
         ])
 
+    # start websocket server
+    p_websocket_server = mp.Process(name='ws',
+                                    target=websocket_server.start,
+                                    args=(exchange.get_subscriber_queue(EventTypes.BLOCK_VALID),))
+    p_websocket_server.start()
+
+    # connect to tendermint event stream
+    p_websocket_client = mp.Process(name='ws_client',
```

Comment:
`ws_to_tendermint`? This would make easier to know what the process is doing when doing a `top` from the terminal.